--- qtcore/src/corelib/global/qlogging.cpp	2021-12-07 05:51:35.000000000 -0800
+++ qtcore/src/corelib/global/qlogging.cpp	2021-12-19 01:18:43.471600044 -0800
@@ -101,16 +101,6 @@
 #include <emscripten/emscripten.h>
 #endif
 
-#if QT_CONFIG(regularexpression)
-#  ifdef __UCLIBC__
-#    if __UCLIBC_HAS_BACKTRACE__
-#      define QLOGGING_HAVE_BACKTRACE
-#    endif
-#  elif (defined(__GLIBC__) && defined(__GLIBCXX__)) || (__has_include(<cxxabi.h>) && __has_include(<execinfo.h>))
-#    define QLOGGING_HAVE_BACKTRACE
-#  endif
-#endif
-
 #if QT_CONFIG(slog2)
 extern char *__progname;
 #endif
